Sources/Extensions/SwiftStdlib/StringExtensions.swift

```diff
@@ -401,7 +401,8 @@ public extension String {
 	///
 	/// - Returns: The reversed string.
 	public func reversed() -> String {
```

Comment:
We should remove this one too. We can just use `String(str.reversed())` if we even need `String` typing